modules/3d/src/ptcloud/ptcloud_utils.cpp

```diff
@@ -0,0 +1,162 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "../precomp.hpp"
+#include "ptcloud_utils.hpp"
+
+
+namespace cv {
+
+void
+_getMatFromInputArray(InputArray input_pts, Mat &mat, int arrangement_of_points, bool clone_data)
```

Comment:
_getMatFromInputArray -> _getPointsMatFromInputArray ?

to avoid using of generic name.

Reply:
fixed.